# databases_in_molecular_biology

### *Molecular Biology*

[BiGGs models](http://bigg.ucsd.edu/): gold-standard genome-scale models

[BioCyc](https://biocyc.org/): pathway/genome databases

[Biomodels](https://www.ebi.ac.uk/biomodels/): A database of models that are used in biology

[cBioPortal](https://www.cbioportal.org/): portal that allows you to explore cancer genomics data visually 

[Database Commons](https://ngdc.cncb.ac.cn/databasecommons/database/id/164): a database of biological databases

[Dunbrack backbone-dependent rotamer library](http://dunbrack.fccc.edu/lab/bbdep2010): provides side-chain conformations based on backbone dihedral angles, enhancing the accuracy of protein design and modeling

[ENCODE](https://www.encodeproject.org/): human regulatory DNA

[Ensembl](https://useast.ensembl.org/index.html): genome data for vertebrates and model organisms

[Gene expression omnibus](https://www.ncbi.nlm.nih.gov/geo/): repository for gene expression datasets

[GotEnzymes](https://metabolicatlas.org/gotenzymes): database of 25M+ $k_{cat}$ measurements of how fast specific enzymes work

[GPCRdb](https://gpcrdb.org/) – focused on GPCRs

[GWAS Catalog](https://www.ebi.ac.uk/gwas/): database of SNP-trait associations

[HGNC (HUGO Gene Nomenclature Committee)](https://www.genenames.org/) – official gene names for human receptors

[HuBMAP](https://hubmapconsortium.org/): modeling the human body at the single cell level

[Human Cell Atlas](https://www.humancellatlas.org/): mapping and annotating every cell in humans

[Human Metabolome Database](https://www.hmdb.ca/): comprehensive database on human metabolites, biomarkers, quantitative

[IEDB (Immune Epitope Database)](https://www.iedb.org/): catalogues experimentally verified immune epitopes (2M of them), which are fragments of antigens recognized by B cells, T cells, and/or MHC molecules 

[IUPHAR/BPS Guide to Pharmacology](https://www.guidetopharmacology.org/) – curated list of receptors, ligands, drug targets

[KEGG](https://www.genome.jp/kegg/pathway.html): models molecular interactions, pathways like modeling

[Metabolic map](https://ecocyc.org/overviewsWeb/celOv.shtml?orgid=ECOLI): metabolic map of E-Coli and others, eventually will have all humans

[OpenHumans](https://www.openhumans.org/explore-share/): platform for sharing data on many different topics- lots of microbiome, genetics, variants, and viral databases

[OpenWetWare.org](http://OpenWetWare.orghttps://openwetware.org/wiki/Protocols): experimental protocol information

[ORF finder](https://www.bioinformatics.org/sms2/orf_find.html): searches for open reading frames (ORFs) in the DNA sequence you enter

[parts.igem.org](http://parts.igem.orghttps://parts.igem.org/Main_Page): standard biological parts, which actually aren’t very standardizes and need to be made engineering-friendly

[PathBank](https://pathbank.org/): biochemical reactions and interactions within cells

[PDB](https://www.rcsb.org/): protein data bank

[PHASTER](https://www.notion.so/Molecular-Biology-1cd5b5e4de3180feb38fc9eb3b230fc9?pvs=21), [PHASTEST](https://phastest.ca/) [VIBRANT](https://github.com/AnantharamanLab/VIBRANT): phage sequence databases

[PhysiCell](https://physicell.org/): virtual laboratory- agent based modeling of cells

[Physiome](https://models.physiomeproject.org/welcome): models organ and tissue function, including circulation, respiration, muscle dynamics (software: https://physiomeproject.org/software)

[Reactome](https://reactome.org/): molecular interactions of cellular processes, pathways

[Saccharomyces Genome Database (SGD)](https://www.yeastgenome.org/): 12M base pairs of yeast DNA sequence and the annotation of over 6k genes + thousands of experiments

[STRING](https://string-db.org/): protein-protein interaction networks, used to identify Ras-associated human proteins 

[TCGA (The Cancer Genome Atlas Project)](https://www.cancer.gov/ccg/research/genome-sequencing/tcga): molecularly characterized over 20,000 primary cancer and matched normal samples spanning 33 cancer type 

[Whole cell model of Myocoplasma genitalium](https://simtk.org/projects/wholecell/): a whole cell model of a living organism, like the worm 

[WikiPathways (NetPath)](https://www.wikipathways.org/authors/NetPath.html): signal transduction pathways in human cells

[Uniprot](https://www.uniprot.org/): protein sequence and functional information, receptors

[uORFdb](https://bioinformatics.uni-muenster.de/tools/uorfdb/index.pl?): upstream ORFs for genetic editing

[VDJdb](https://vdjdb.cdr3.net/): database of T-cell receptor (TCR) sequences with known antigen specificities

[Virtual Physiological Human](https://www.vph-institute.org/): simulates entire-body physiological interactions, up to disease progression

### *Biotech*

[Drugs@FDA](https://www.accessdata.fda.gov/scripts/cder/daf/index.cfm): FDA approved drugs by month 

[Enamine REAL Space](https://enamine.net/compound-collections/real-compounds/real-space-navigator): 38B drug combinations

- Is REAL space limited by the Lipinski rule?

[FDA Purple Book](https://purplebooksearch.fda.gov/about): approved biologics and biosimilars

[Kegg](https://www.genome.jp/kegg/drug/): drug database 

### *Tools*

[aRNA amplification](https://www.thermofisher.com/order/catalog/product/AM1751): linear amplification of single-cell RNA (what is logarithmic?)

[RUM (RNA-seq Unified Mapper)](https://github.com/itmat/rum): alignment tool that maps RNA-seq reads to the genome (very old, don’t recommend using)

[Oakvar](http://agingkills.eu:8081/index.html#): collection of genome and variant annotation tools